Tests/DatadogTests/Datadog/RUM/Integrations/CrashReportReceiverTests.swift

```diff
@@ -498,6 +498,13 @@ class CrashReportReceiverTests: XCTestCase {
             && sendRUMErrorEvent.model.view.id == lastRUMViewEvent.view.id,
             "The `RUMErrorEvent` sent must be linked to the same RUM Session as the last `RUMViewEvent`."
         )
+
+        DDAssertJSONEqual(
+            AnyEncodable(sendRUMErrorEvent.context?.contextInfo),
+            AnyEncodable(lastRUMViewEvent.context?.contextInfo),
+            "The `RUMErrorEvent` sent must be include the context of the last `RUMViewEvent`."
+        )
```

Comment:
suggestion/ Are we sure both aren't `nil`, making this assertion not sufficient? We could consider adding `XCAssertNotNil` before or using `XCTUnwrap` to get non-optional.